utils/releaser/src/ReleaseWorker/ReleaseCandidate/BuildProjectBaseOnHeimdallReleaseWorker.php

```diff
@@ -1,45 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Shopsys\Releaser\ReleaseWorker\ReleaseCandidate;
-
-use PharIo\Version\Version;
-use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
-use Shopsys\Releaser\Stage;
-
-final class BuildProjectBaseOnHeimdallReleaseWorker extends AbstractShopsysReleaseWorker
```

Comment:
I would keep this and ask a developer to check the GitHub Actions status of the project-base.

Reply:
good idea